### DISCUSSION TITLE
web_1           | 2021/07/31 11:17:44 http: TLS handshake error from x.x.x.x:62680: no certificate available for '172.28.0.5

### DISCUSSION THREAD

**Discussion by khanh78180** [opengisch/QField#2455] [Q&A]
Hi,

I'm testing Qfieldcloud and I have an issue with the following message when trying to connect to the URL https://x.x.x.x

docker-compose up -d --build
rchestrator_1  | {"message": "Dequeue QFieldCloud Jobs from the DB", "time": "2021-07-31T10:11:21.702032"}
web_1           | 2021/07/31 11:17:44 http: TLS handshake error from x.x.x.x:62680: no certificate available for '172.28.0.5'
web_1           | 2021/07/31 11:17:44 http: TLS handshake error from x.x.x.x:62078: no certificate available for '172.28.0.5'

Is there someone has already encounter this issue ?
How can I solve it ?

Thank for help.
Regards

**Comment by mbernasocchi** [comment]
@suricactus any suggestions?

**Comment by suricactus** [comment]
Hello, @khanh78180 . What I can recommend is copying `docker-compose.override.local.yml` as `docker-compose.override.yml` and using this configuration for local development.

**Comment by khanh78180** [comment]
Hi Suricactus,

Thank for your advice
I'll try it

**Comment by marioba** [comment]
In order to provide a clearer and more manageable discussion forum and to have all discussions about QField and related products (i.e. QFieldCloud and QFieldSync) in a single place, we transferred all the discussions from the QFieldCloud to the [QField repository](https://github.com/opengisch/QField/discussions).
